Rename "What can I do about Bufferbloat?"

richb-hanover opened the pull request:
(from "What to do about Bufferbloat?") and also

Update Make Wi-Fi Fast home page with new information about LEDE wifi fixes and Toke's paper

[This time for sure...]

tohojo commented on content/bloat/wiki/What_can_I_do_about_Bufferbloat.md:
```diff
@@ -1,14 +1,17 @@
-Test](http://dslreports.com/speedtest) or run one of the [Tests for Bufferbloat](Tests_for_Bufferbloat.md) to see if it's present.
+traffic on your network. 
+Use the [DSLReports Speed Test](http://dslreports.com/speedtest) 
+or run one of the [Tests for Bufferbloat](/bloat/wiki/Tests_for_Bufferbloat.md) to see if it's present.
```
Why the change of link style?

tohojo commented on content/bloat/wiki/What_can_I_do_about_Bufferbloat.md:
```diff
@@ -18,7 +21,7 @@ will need to find a way to fix the **router.**
 ## How Can I Tell if My Router Has Bufferbloat?
 
--   Use [DSL Reports Speed Test](http://dslreports.com/speedtest) or any of the other tests on [Tests for Bufferbloat](Tests_for_Bufferbloat.md)
+-   Use [DSL Reports Speed Test](http://dslreports.com/speedtest) or any of the other tests on [Tests for Bufferbloat](/bloat/wiki/Tests_for_Bufferbloat/)
```
Also changed link style (to absolute path)

tohojo commented on content/cerowrt/wiki/What_t.md:
```diff
@@ -8,4 +8,4 @@ What t
 
 Redirect to
-http://www.bufferbloat.net/projects/cerowrt/wiki/What\_to\_do\_about\_Bufferbloat
+http://www.bufferbloat.net/projects/bloat/wiki/What\_can\_I\_do\_about\_Bufferbloat
```
Not sure what this page is supposed to be doing?

tohojo commented on content/make-wifi-fast/wiki/index.md:
```diff
@@ -1,13 +1,34 @@
+As of early 2017, we have achieved many of these improvements, specifically a decrease of latency by at least an order of magnitude, with fair sharing of air time across fast and slow devices.
+
+* Working software is available in <a href="https://lede-project.org" target="blank">LEDE firmware</a> that runs on off-the-shelf routers, x86 boxes, and embedded systems.
+* An academic paper has been published to describe the current state of the working software, 
```
Hasn't technically been published yet; only the preprint...

tohojo commented on layouts/index.html:
```diff
@@ -14,29 +14,29 @@
 
-              <li><strong><a href="/bloat/wiki">The Bufferbloat Project</a></strong> 
-              has largely addressed
+              <li><strong><a href="http://localhost:1313/projects/bloat/wiki">
```
As is this

tohojo commented on layouts/index.html:
```diff
@@ -14,29 +14,29 @@
-            <ul>
-              <li><strong><a href="/bloat/wiki/What_to_do_about_Bufferbloat/">What to do about Bufferbloat?</a></strong> If you have bad latency/lag, or someone has told you there is Bufferbloat in your network. This page lists several steps that you can take to measure the bloat in your network, and how make solve it completely.</li>
+              <ul>
+              <li><strong><a href="http://localhost:1313/projects/bloat/wiki/What_can_I_do_about_Bufferbloat/">What Can I Do About Bufferbloat?</a></strong> 
```
This is certainly broken

tohojo commented on layouts/index.html:
```diff
@@ -14,29 +14,29 @@
+              delay from competing traffic on a bottleneck link to the order of 20 msec.</li>
 
-              <li><strong><a href="/make-wifi-fast/wiki">
+              <li><strong><a href="http://localhost:1313/projects/make-wifi-fast/wiki">
```
And this

tohojo commented on layouts/index.html:
```diff
@@ -14,29 +14,29 @@
-              to Wi-Fi stacks is bearing fruit, and is available in 
-              <a href="https://lede-project.org" target="blank">LEDE firmware.</a> </li>
+              As of early 2017, our efforts to remove queueing latency and add Airtime Fairness 
+              to Wi-Fi stacks have borne fruit. See the <a href="http://localhost:1313/projects/make-wifi-fast/wiki/">Make Wi-Fi Fast Status</a> page.
```
And this...

tohojo commented on content/bloat/wiki/What_can_I_do_about_Bufferbloat.md:
```diff
@@ -1,14 +1,17 @@
+lastmod: 2017-03-10T09:20:12
 type: wiki
+aliases:
+    - /projects/bloat/what_to_do_about_bufferbloat.md
```
There shouldn't be an .md suffix on the alias, and I think it is case sensitive...

richb-hanover commented:
Hi Toke,

1) I don't understand the rules/ethics/politics of publishing a link to a preprint. I look to your guidance, but we shouldn't hide our light under a bushel...
2) I changed several sentences to start each on a new line. It makes detecting editing changes much easier to see (in the future).
3) Fixed the absolute links that could be relative.
4) The "What_t.md" file is a holdover from the Redmine wiki. (Its title started out to be "What to do about Bufferbloat", but something failed, and I didn't see how to modify the title.) I've deleted it.
5) ... http://localhost:1313 <http://localhost:1313/> ... Stupid. Fixed.
6) Fixed many aliases to the renamed "What can I do" - I think their case-sensitivity and suffix are correct for links that are in the wild. (Is it possible to browse the error.log file on the bufferbloat.net <http://bufferbloat.net/> server?)

Thanks for your patience.

Rich

> 
> In content/bloat/wiki/What_can_I_do_about_Bufferbloat.md <https://github.com/tohojo/bufferbloat-net/pull/31#discussion_r105567482>:
> 
> >  
>  Bufferbloat is high latency (or lag) that occurs when there's other
> -traffic on your network. Use the [DSLReports Speed
> -Test](http://dslreports.com/speedtest) or run one of the [Tests for Bufferbloat](Tests_for_Bufferbloat.md) to see if it's present.
> +traffic on your network. 
> +Use the [DSLReports Speed Test](http://dslreports.com/speedtest) 
> +or run one of the [Tests for Bufferbloat](/bloat/wiki/Tests_for_Bufferbloat.md) to see if it's present.
> Why the change of link style?
> 
> In content/bloat/wiki/What_can_I_do_about_Bufferbloat.md <https://github.com/tohojo/bufferbloat-net/pull/31#discussion_r105567488>:
> 
> > @@ -18,7 +21,7 @@ will need to find a way to fix the **router.**
>  
>  ## How Can I Tell if My Router Has Bufferbloat?
>  
> --   Use [DSL Reports Speed Test](http://dslreports.com/speedtest) or any of the other tests on [Tests for Bufferbloat](Tests_for_Bufferbloat.md)
> +-   Use [DSL Reports Speed Test](http://dslreports.com/speedtest) or any of the other tests on [Tests for Bufferbloat](/bloat/wiki/Tests_for_Bufferbloat/)
> Also changed link style (to absolute path)
> 
> In content/cerowrt/wiki/What_t.md <https://github.com/tohojo/bufferbloat-net/pull/31#discussion_r105567493>:
> 
> > @@ -8,4 +8,4 @@ What t
>  ======
>  
>  Redirect to
> -http://www.bufferbloat.net/projects/cerowrt/wiki/What\_to\_do\_about\_Bufferbloat
> +http://www.bufferbloat.net/projects/bloat/wiki/What\_can\_I\_do\_about\_Bufferbloat
> Not sure what this page is supposed to be doing?
> 
> In content/make-wifi-fast/wiki/index.md <https://github.com/tohojo/bufferbloat-net/pull/31#discussion_r105567502>:
> 
> > +and hardware. 
> +
> +**Our Manifesto - Wi-Fi does not need to be slow!**
> +
> +The hardware now available for Wi-Fi can accomplish tremendous performance, but it is hobbled by software designs that guarantee high latency under load. 
> +This, in turn, dramatically lowers performance in real-world settings (multiple users, home routers, commercial access points) leading to the *myth* that "Wi-Fi is always slow."
> +
> +We believe that the same sort of systems thinking that went on in the Bufferbloat Project can lead to 
> +performance improvements of an order of magnitude or more in Wi-Fi.
> +
> +## Current Status
> +
> +As of early 2017, we have achieved many of these improvements, specifically a decrease of latency by at least an order of magnitude, with fair sharing of air time across fast and slow devices.
> +
> +* Working software is available in <a href="https://lede-project.org" target="blank">LEDE firmware</a> that runs on off-the-shelf routers, x86 boxes, and embedded systems.
> +* An academic paper has been published to describe the current state of the working software, 
> Hasn't technically been published yet; only the preprint...
> 
> In layouts/index.html <https://github.com/tohojo/bufferbloat-net/pull/31#discussion_r105567513>:
> 
> >  
> -              <li><strong><a href="/bloat/wiki">The Bufferbloat Project</a></strong> 
> -              has largely addressed
> +              <li><strong><a href="http://localhost:1313/projects/bloat/wiki">
> As is this
> 
> In layouts/index.html <https://github.com/tohojo/bufferbloat-net/pull/31#discussion_r105567519>:
> 
> > @@ -14,29 +14,29 @@
>                projects provide a webspace for addressing chaotic and laggy
>                network performance. We have a number of projects in flight:</p>
>  
> -            <ul>
> -              <li><strong><a href="/bloat/wiki/What_to_do_about_Bufferbloat/">What to do about Bufferbloat?</a></strong> If you have bad latency/lag, or someone has told you there is Bufferbloat in your network. This page lists several steps that you can take to measure the bloat in your network, and how make solve it completely.</li>
> +              <ul>
> +              <li><strong><a href="http://localhost:1313/projects/bloat/wiki/What_can_I_do_about_Bufferbloat/">What Can I Do About Bufferbloat?</a></strong> 
> This is certainly broken
> 
> In layouts/index.html <https://github.com/tohojo/bufferbloat-net/pull/31#discussion_r105567524>:
> 
> >  
> -              <li><strong><a href="/make-wifi-fast/wiki">
> +              <li><strong><a href="http://localhost:1313/projects/make-wifi-fast/wiki">
> And this
> 
> In layouts/index.html <https://github.com/tohojo/bufferbloat-net/pull/31#discussion_r105567526>:
> 
> >                The Make-Wi-Fi-Fast Project,</a></strong> with many of the
>                same team members as the Bufferbloat project, intends to improve
>                Wi-Fi's speed and use of the spectrum by inserting CoDel/fq_codel
>                into the Wi-Fi queues, and actively measuring the power required
>                for successful transmission, in order to minimize contention and
>                interference on the RF channel. 
> -              As of early 2017, our efforts to remove queues, and add Airtime Fairness 
> -              to Wi-Fi stacks is bearing fruit, and is available in 
> -              <a href="https://lede-project.org" target="blank">LEDE firmware.</a> </li>
> +              As of early 2017, our efforts to remove queueing latency and add Airtime Fairness 
> +              to Wi-Fi stacks have borne fruit. See the <a href="http://localhost:1313/projects/make-wifi-fast/wiki/">Make Wi-Fi Fast Status</a> page.
> And this...
> 
> In content/bloat/wiki/What_can_I_do_about_Bufferbloat.md <https://github.com/tohojo/bufferbloat-net/pull/31#discussion_r105567570>:
> 
> >  type: wiki
> +aliases:
> +    - /projects/bloat/what_to_do_about_bufferbloat.md
> There shouldn't be an .md suffix on the alias, and I think it is case sensitive...
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/tohojo/bufferbloat-net/pull/31#pullrequestreview-26453840>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ABC1EsfcmQHEfzG6I7mApLy0BPxKveNIks5rlFDxgaJpZM4MapUH>.
>

tohojo commented:
For the paper I would just write something like "An academic paper describing the current state of the working software has been submitted for publication; a preprint is available at..."

I'll email you the log file.

richb-hanover commented:
OK. I'll update the description in my branch. Does it otherwise look OK?

Thanks.

Rich

> On Mar 17, 2017, at 7:57 AM, Toke Høiland-Jørgensen <notifications@github.com> wrote:
> 
> For the paper I would just write something like "An academic paper describing the current state of the working software has been submitted for publication; a preprint is available at..."
> 
> I'll email you the log file.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/tohojo/bufferbloat-net/pull/31#issuecomment-287336143>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ABC1EsK4WksuZJPB9u-eeYldUw8EdYI_ks5rmnU7gaJpZM4MapUH>.
>

richb-hanover commented:
I just pushed that update. Please check.

tohojo commented:
There are some weird changes to cerowrt/issues/216.md - and a large diff of a .csv file. Why is that?

Also, aliases should not have .md extensions; they operate solely on the publicly visible URL space...

richb-hanover commented:
* There don't seem to be any urls-with-.md in the wild, so I have removed those.  

* The 216.md file has wacky email addresses (if anyone attempted to use them) that were tripping up LinkChecker. Same for the .csv file. But I care more about publishing these changes than being error-free.

* I added a link to the Untangle firewall (it has fq_codel) and fixed lots of wording/typos.

dtaht commented:
toke is on a big boat somewhere, and the division of labor here is unclear. But LGTM.

richb-hanover commented:
Hey! The magic works!
